File: Developer/DeveloperTools/YJTimeline/YJTimeline/ViewController.m
//
//  ViewController.m
//  YJTimeline
//
//  Created by CISDI on 2019/1/19.
//  Copyright © 2019 YJCocoa. All rights reserved.
//

#import "ViewController.h"
#import <YJCocoa/YJTimeline.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [YJTimeline addSteps:@"ViewController-viewDidLoad"];
}

- (void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear:animated];
    [YJTimeline addSteps:@"ViewController-viewDidLoad"];
}

@end

File: Demo/Swift/DeveloperTools/YJLeaks/YJLeaks/ViewController.swift
//
//  ViewController.swift
//  YJLeaks
//
//  Created by 阳君 on 2019/6/17.
//  Copyright © 2019 YJCocoa. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.view.backgroundColor = UIColor.white
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        self.navigationController?.pushViewController(YJLeaksViewController(), animated: true)
    }
    
    
}

